perceval/runtime/remote_job.py

```diff
@@ -97,8 +116,12 @@ def execute_async(self, *args, **kwargs):
                 if self._job_context is None:
                     self._job_context = {}
                 self._job_context["mapping_delta_parameters"] = self._delta_parameters
+            kwargs = self._handle_unnamed_params(args, kwargs)
             kwargs['job_context'] = self._job_context
-            self._id = self._fn(*args, **kwargs)
+            self._payload['job_name'] = self._name
+            self._payload['payload'].update(kwargs)
```

Comment:
It supposes that the payload has been created by a processor before (maybe add a docstring ?)